ProjectSettings/GraphicsSettings.asset

```diff
@@ -36,7 +36,7 @@ GraphicsSettings:
   m_PreloadedShaders: []
   m_PreloadShadersBatchTimeLimit: -1
   m_SpritesDefaultMaterial: {fileID: 10754, guid: 0000000000000000f000000000000000, type: 0}
-  m_CustomRenderPipeline: {fileID: 11400000, guid: 1f2d4bf0803063e95843047ec21d3304, type: 2}
```

Comment:
This setting cause weird behavior for me...

![image](https://github.com/grupacosmo/cosmobot/assets/49162745/408f7c55-5a47-478a-97f3-8d096f93ca2f)

But this is probably my local issue so will investigate this for myself later

Reply:
Revert this change before merge

Reply:
Reverted